### PR TITLE
Bump CoreDNS manifests to latest stable version 1.8.3

### PR DESCRIPTION
**Pull request by dntosas**
In this commit, we update manifest files of CoreDNS. We also do a small
refactoring on yaml blocks to match the upstream source and to make
updating and reviewing procedures easier on the future.

Signed-off-by: dntosas <ntosas@gmail.com>

### CODE REVIEW

File: upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template [hunk 1]
```diff
@@ -219,8 +179,8 @@ spec:
           failureThreshold: 5
         readinessProbe:
           httpGet:
-            path: /health
-            port: 8080
+            path: /ready
+            port: 8181
```
Comment (dntosas) [inline upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template:183]: this seems not to be working

checking and if needed we will also push a PR on upstream

Comment (dntosas) [reply upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template]: oh ok, needed `ready` plugin on configuration

fixed

File: upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template [hunk 2]
```diff
@@ -219,8 +180,8 @@ spec:
...
```
Comment (dntosas) [inline upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template:184]: could that be a breaking change for some setup that use custom configuration? WDYT?

`ready` plugin is needed, that we add in our default config file ^^

Comment (olemarkus) [reply upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template:184]: I cannot imagine this will be a problem.

**Review by olemarkus — APPROVED** [review]

File: upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template (hunk 2 above)
Comment (dntosas) [reply upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template:184]: @olemarkus there were some users that use custom coreDNS config reporting this to be a breaking change for their setups. maybe good idea of adding a release note to guide of needed actions in such cases? WDYT?

**Pull request merged**